### DISCUSSION TITLE
How do downloads work?

### DISCUSSION THREAD

**Discussion by nonetrix** [Shabinder/SpotiFlyer#1044] [Q&A]
I have seen a lot of "Spotify downloaders" that just search for the song on YouTube Music, that does work but isn't 100% full proof. Does this somehow get around Spotify DRM, or use APIs somehow? Is there any chance Spottily could take this down if they wanted?

Also thanks for this :D

**Comment by Shabinder** [comment]
> I have seen a lot of "Spotify downloaders" that just search for the song on YouTube Music, that does work but isn't 100% full proof. Does this somehow get around Spotify DRM, or use APIs somehow? Is there any chance Spottily could take this down if they wanted?
> 
> Also thanks for this :D

This also matches the songs using parameters on other sources like JioSaavn and youtube and downloads the best quality match found
